app/gui2/src/components/GraphEditor/widgets/WidgetSelection.vue

```diff
@@ -136,11 +139,21 @@ watch(selectedIndex, (_index) => {
 </script>
 
 <script lang="ts">
+function hasBooleanTagValues(parameter: SuggestionEntryArgument): boolean {
+  if (parameter.tagValues == null) return false
```

Comment:
This whole function could be made simpler by creating some sorts of  `arrayEquals` utility and using it here.